stacks/dlrs/releasenote.md

```diff
@@ -43,6 +63,10 @@ The official Deep Learning Reference Stack Docker images are hosted at: https://
 
  * Pull from the [Eigen version](https://hub.docker.com/r/clearlinux/stacks-dlrs-oss/)
  * Pull from the [Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks-dlrs-mkl/)
+ * Pull from the [PyTorch version](https://hub.docker.com/r/clearlinux/stacks...)
+ * Pull from the [PyTorch with Intel MKL version](https://hub.docker.com/r/clearlinux/stacks...)
+ * Pull from the [PyTorch with Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks...)
```

Comment:
Just being noisy but are those links valid?

Reply:
They need review before this can be merged.

Reply:
+1 looks good to me, will provide you the new links asap.

Reply:
Links updated, ready to go.